server/lib/tuist_web/live/qa_settings_live.html.heex

```diff
@@ -47,17 +85,17 @@
             placeholder={gettext("argument-group-name")}
             type="basic"
           />
-          <.text_input
+          <.text_area
             field={@add_launch_argument_form[:description]}
             label={gettext("Description")}
             placeholder={gettext("Launch argument group description")}
-            type="basic"
+            show_character_count={false}
```

Comment:
when the phoenix view is rerendered, such as when validating, the modal loses focus and basically becomes unusable. To get around this, we're not showing the character count, although it's in the design. This issue only occurs in modals. Something to look into in the future.